EventListener/UserContextSubscriber.php

```diff
@@ -121,6 +133,20 @@ public function onKernelRequest(GetResponseEvent $event)
     }
 
     /**
+     * Tests if $request is an anonymous request or not.
+     *
+     * For backward compatibility reasons, true will be returned if no anonymous request matcher was provided.
+     *
+     * @param Request $request
+     *
+     * @return bool
+     */
+    private function isAnonymous(Request $request)
+    {
+        return isset($this->anonymousRequestMatcher) ? $this->anonymousRequestMatcher->matches($request) : false;
```

Comment:
lets remove the isset here and just say $this->anonymousRequestMatcher ? $this-... or if you don't like this null !== $this->anonymousRequestMatcher ?